specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-01-01/examples/FrontdoorCreate.json

```diff
@@ -216,7 +218,7 @@
                     "priority": 1
                   },
                   {
-                    "privateLinkResourceId": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/privateLinkServices/pls1",
+                    "privateLinkAlias": "SQLSERVER.d84e61f0-0870-4d24-9746-7438fa0019d1.westus2.azure.privatelinkservice",
```

Comment:
same comment

Reply:
This is not a production alias, this was generated to use for the examples